File: Q08/main.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cin>>n;
    int arr[n];
    for(int i=0;i<n;i++){
        cin>>arr[i];
    }

    int i=0;    //count of unique elements 
    for(int j=1;j<n;j++){
        if(arr[j]!=arr[i]){
            i++;
            arr[i]=arr[j];
        }
    }
    for(int k=0;k<=i;k++){
        cout<<arr[k];
    }
    return 0;
}

File: Q08/main.py
n=int(input())
arr=[]

for i in range(n):
    arr.append(int(input()))

i=0
for j in range(1,n):
    if arr[j]!=arr[i]:
        i+=1
        arr[i]=arr[j]

for k in range(i+1):
    print(arr[k],end=" ")